monitor/service.go

```diff
@@ -200,6 +196,12 @@ func (m *Monitor) Statistics(tags map[string]string) ([]*Statistic, error) {
 					}
 					statistic.Values[kv.Key] = f
 				})
+			case "references":
+				s := subKV.Value.(*expvar.Int).String()
+				count, _ := strconv.ParseInt(s, 10, 32)
+				if count < 2 {
+					influxdb.DeleteStatistics(kv.Key)
```

Comment:
Yes, the reason for the delete - perhaps we should call it close - is to allow the monitor to see the final batch of updates